doc/manual.md

```diff
@@ -24,48 +24,35 @@ upon the abstract, and a tool for translating concepts into *Signal
 Transition Graphs* (STGs). These
 are commonly used for the specification, verification and synthesis of
 asynchronous control circuits in the academic community, and they are
-supported by multiple EDA tools, such as <span><span
-style="font-variant:small-caps;">Petrify</span></span>,
-<span><span
-style="font-variant:small-caps;">Mpsat</span></span>,
-<span><span
-style="font-variant:small-caps;">Versify</span></span>,
-<span><span
-style="font-variant:small-caps;">Workcraft</span></span>,
-and others. The aim of this tool is to design and debug concepts, and
-then translate them to STGs, where they can be used with these tools.
+supported by multiple EDA tools, such as Petrify, Mpsat, Versify,
+Workcraft, and others. The aim of this tool is to design and debug
+concepts, and then translate them to STGs, where they can be used with
+these tools.
 
 The latest version of this tool, and this manual, can be found [in the
 GitHub repo](https://github.com/tuura/concepts). Any bugs or issues
 found with the tool can be reported here. This tool is also distributed
-as a back-end tool for <span><spanstyle="font-variant:small-caps;">Workcraft</span></span>. This version
+as a back-end tool for orkcraft. This version
 of the concepts tool will be the latest version that works correctly
```

Comment:
Orkcraft sounds great, but I'm not sure orks actually have any use for asynchronous concepts... :thinking: